redislock.go

```diff
@@ -54,8 +62,15 @@ func New(client RedisClient) *Client {
 // Obtain tries to obtain a new lock using a key with the given TTL.
 // May return ErrNotObtained if not successful.
 func (c *Client) Obtain(ctx context.Context, key string, ttl time.Duration, opt *Options) (*Lock, error) {
-	token := opt.getToken()
+	return c.ObtainMany(ctx, []string{key}, ttl, opt)
+}
 
+// ObtainMany tries to obtain new locks using keys with the given TTL.
```

Comment:
minor: don't you think `ObtainMulti` sounds better? also, more consistent with go stdlib

Reply:
Renamed